lib/src/validations.dart

```diff
@@ -0,0 +1,15 @@
+// Determine if a String is digit
+bool isDigit(String s) {
```

Comment:
La cadena debería estar compuesta solo de números, podrías tratar de convertir la cadena completa, algo como https://stackoverflow.com/questions/24085385/checking-if-string-is-numeric-in-dart 
Así no tienes que iterar sobre cada posición

Reply:
ok ya lo reviso

Reply:
el problema de usar eso que me enviaste, es que la funcion onError esta deprecated

Reply:
usare try catch

Reply:
No había visto que estaba como deprecated. Puedes usar:
```dart
return double.tryParse(s) != null;
```